### PR TITLE
This is a few revisions to Section 3

### PR DESCRIPTION
**Pull request by jhdalek55**
I made a few changes here, as well as recommendation about the scenarios

### CODE REVIEW

**Comment by jhdalek55** [comment]
I have updated Section 4 now.

File: design.tex [hunk 1]
```diff
@@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
 registration form \yanyan{cite url or show screenshot} at the 
 clearinghouse. The clearinghouse website shows 
 a list of available sensors, and each of their required accuracy 
-and access frequency. Aftering filling out this form, the researcher 
+and access frequency.\lois(does it show the "sensors" apart from the devices?) After filling out this form, the researcher
```
Comment (yyzhuang) [inline design.tex:159]: What does this mean?

Comment (jhdalek55) [inline design.tex:159]: You say it "shows a s list of available sensors" as opposed to a "list of
available devices." So I wanted to know if the clearinghouse literally
offers experimenters "200 GPS" as opposed to 200 phones that have that
capability. It's not a huge point, but it does slightly change the
description if what the experimenter is choosing from...

Comment (yyzhuang) [inline design.tex:159]: I see what the confusion is now. Thanks!

Comment (jhdalek55) [inline design.tex:159]: No problem.

Lois

On Wed, Nov 11, 2015 at 05:42 PM, Yanyan Zhuang wrote:

In design.tex 
https://github.com/yyzhuang/mobisys16-sensibility/pull/2#discussion_r44598015 
:

> @@ -158,64 +156,62 @@ \subsubsection{Clearinghouse}\label{sec-ch}
>  registration form \yanyan{cite url or show screenshot} at the 
> clearinghouse. The clearinghouse website shows  a list of available 
> sensors, and each of their required accuracy -and access frequency. 
> Aftering filling out this form, the researcher +and access 
> frequency.\lois(does it show the "sensors" apart from the devices?) 
> After filling out this form, the researcher

I see what the confusion is now. Thanks!

—
Reply to this email directly or view it on GitHub 
https://github.com/yyzhuang/mobisys16-sensibility/pull/2/files#r44598015 
.

File: design.tex [hunk 2]
```diff
@@ -256,19 +252,18 @@ \subsubsection{Experiment Manager}\label{sec-emt}
...
-In the following, we present several usage scenarios to
+In the following section, we present several usage scenarios to
 demonstrate how device owners opt in as volunteers, and how
-researchers do experiments on devices without compromising device
+researchers are able to to conduct experiments on devices without compromising device
 owners' privacy.
 
 \subsection{Testbed Usage Scenarios}\label{sec-scenario}
-%\yanyan{should this section go after policies?}
+%\yanyan{should this section go after policies?} \lois(After reviewing these this morning, I am not sure you need these scenarios at all. They seem to repeats examples we have already seen in previous sections. We already saw the procedures for Bob to fill out the form or for Alice to download the app. There might be a few bits and pieces of new information here that we could work into something else?such as the twbut I don't think we need these scenarios as a separate section.)
```
Comment (yyzhuang) [inline design.tex:261]: I feel the same way...

File: policy.tex [hunk 3]
```diff
@@ -117,13 +114,13 @@ \subsubsection{Sensor Access from the Python Sandbox}
 %files, send data through the network, etc., from Alice's device. 
 %As in Section~\ref{sec-repy}, the Repy sandbox does not include 
 %calls to access sensors. But 
-To allow an experimenter's code to access sensors such as 
-GPS, WiFi, Bluetooth, accelerometer, cellular network, etc., we  
-need access to native Android code. We first implemented a set of sensor functions using
-Java in the native portion of the Sensibility Testbed app. The Repy 
+To allow an experimenter's code to access sensors, such as 
+GPS, WiFi, Bluetooth, accelerometer, and cellular network, we  
+need access to native Android code. \lois(I think I already asked this, but does this only run on Android devices? If so, I think that needs to be mentioned. Also, when you say "we need access to Android code, who is "we?" The researchers? The user? Repy management?) We first implemented a set of sensor functions using
```
Comment (yyzhuang) [inline policy.tex:119]: I added Android in intro.

File: policy.tex [hunk 4]
```diff
@@ -117,13 +114,13 @@ \subsubsection{Sensor Access from the Python Sandbox}
...
 %As in Section~\ref{sec-repy}, the Repy sandbox does not include 
 %calls to access sensors. But 
-To allow an experimenter's code to access sensors such as 
-GPS, WiFi, Bluetooth, accelerometer, cellular network, etc., we  
-need access to native Android code. We first implemented a set of sensor functions using
-Java in the native portion of the Sensibility Testbed app. The Repy 
+To allow an experimenter's code to access sensors, such as 
+GPS, WiFi, Bluetooth, accelerometer, and cellular network, we  
+need access to native Android code. \lois(I think I already asked this, but does this only run on Android devices? If so, I think that needs to be mentioned. Also, when you say "we need access to Android code, who is "we?" The researchers? The user? Repy management?) We first implemented a set of sensor functions using 
+Java in the native portion of the Sensibility Testbed app \lois(Again, who implemented the sensors? And are you referring to the design initially done that is now in the oast? Or the actions taken by a user now) The Repy
```
Comment (yyzhuang) [inline policy.tex:120]: It's in the initial design. Users don't have to do anything.

File: policy.tex (hunk 3 above)
Comment (jhdalek55) [inline policy.tex:119]: OK

File: policy.tex (hunk 4 above)
Comment (jhdalek55) [inline policy.tex:120]: I'll look at this again.

**Pull request closed without merging**